.github/workflows/docs.yml

```
@@ -100,13 +101,25 @@ jobs:
         if: always()
         run: tar -zcvf docs-build.tar.gz doc/_build/html/
 
+      - name: Tar build multiversion
+        if: always()
+        working-directory: ./doc/_build/html/
+        run: tar -zcvf ../../docs-build-multi.tar.gz .
```

nitpick, but why set the working directory to something then perform the operation two levels up? Why not just

```suggestion
        working-directory: ./doc/
        run: tar -zcvf docs-build-multi.tar.gz .
```

---

Probably to remove the two enclosing directories from the archive. But I bet `tar` has options to do that.

---

Ohhhhh, good point

---

Looks like it uses `-C`, just like `make` https://man7.org/linux/man-pages/man1/tar.1.html.